Add instructions to add metrics dashboard for NGINXaaS for Google (#1880)
nginx/documentation: content/nginxaas-google/monitoring/metrics-catalog.md

### NGINX config statistics

> These metrics are collected automatically for all NGINXaaS deployments. No additional NGINX configuration is required.

Reviewer: ```suggestion
All NGINXaaS deployments collect these metrics automatically. No additional NGINX configuration is required.
```

Changed to active voice.

### NGINX connections statistics

> These metrics are collected automatically for all NGINXaaS deployments. No additional NGINX configuration is required.

Reviewer: ```suggestion
All NGINXaaS deployments collect these metrics automatically. No additional NGINX configuration is required.
```

Changed to active voice.

### NGINX SSL statistics

> These metrics are collected automatically when SSL/TLS is configured in your NGINX deployment. To collect SSL metrics, configure SSL certificates and enable HTTPS listeners in your NGINX configuration.

Reviewer: ```suggestion
NGINX automatically collects these metrics when you configure SSL/TLS in your NGINX deployment. To collect SSL metrics, configure SSL certificates and enable HTTPS listeners in your NGINX configuration.
```

### NGINX memory statistics

> These metrics track shared memory zone usage. Memory statistics are collected automatically when you configure zones using the `status_zone` directive or other directives that create shared memory zones.

Reviewer: ```suggestion
These metrics track shared memory zone usage. NGINX automatically collects memory statistics when you configure zones using the `status_zone` directive or other directives that create shared memory zones.
```
